config/environments/production.rb

```diff
@@ -43,7 +43,11 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for nginx
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
+
+  # Exclude SSL on Healthcheck and ping.json calls
+  config.ssl_options = { redirect: { exclude: -> request { request.path.include? '/ping.json/' } } }
```

Comment:
Should it be ```suggestion
  config.ssl_options = { redirect: { exclude: -> request { request.path.include? '/ping.json' } } }
```
These here slashes on the end of the URI look odd  - should these be here? They _look_ like they'll stop it matching because /ping.json is the end of the string, for example

Reply:
Yes, just edited the code to '/ping.json'.